app/package.json

```diff
@@ -18,8 +19,8 @@
     "lint": "eslint --ext .js,.jsx,.ts,.tsx,.astro src",
     "lint:fix": "eslint --ext .js,.jsx,.ts,.tsx,.astro src --fix",
     "lint:commit": "commitlint --edit \"$1\" --verbose",
-    "lint:docs": "markdownlint ../*.md ../docs/*.md ../docs/**/*.md --ignore node_modules",
-    "lint:docs:fix": "markdownlint ../*.md ../docs/*.md ../docs/**/*.md --ignore node_modules --fix",
+    "lint:docs": "markdownlint ../*.md ../docs/*.md ../docs/**/*.md --ignore node_modules --ignore ../CHANGELOG.md",
+    "lint:docs:fix": "markdownlint ../*.md ../docs/*.md ../docs/**/*.md --ignore node_modules --ignore ../CHANGELOG.md --fix",
```

Comment:
Even though we decided to go a different way with the changelog, can we add a commit with just these 2 lines and update the PR title accordingly, please?

I think this will be useful in the future.

Oh, and remove the rest of the changes in this PR as they are not needed anymore.

Thanks!